### PR TITLE
Updating AttachmentSizeValidator (removing Rails 3 support)

### PR DESCRIPTION
**Pull request by lucascaton**
* Rails 3.2 has been removed (#2074)
* V5.0 Release Plan (#2049)

### CODE REVIEW

File: spec/paperclip/validators/attachment_size_validator_spec.rb [hunk 1]
```diff
@@ -158,8 +150,10 @@ def self.should_not_allow_attachment_file_size(size, options = {})
           message: "is invalid. (Between %{min} and %{max} please.)"
       end
 
-      should_not_allow_attachment_file_size 11.kilobytes,
-        message: "is invalid. (Between #{storage_units[5120]} and #{storage_units[10240]} please.)"
+      should_not_allow_attachment_file_size(
+        11.kilobytes,
+        message: "is invalid. (Between 5 KB and 10 KB please.)"
```
Comment (lucascaton) [inline spec/paperclip/validators/attachment_size_validator_spec.rb]: Ew, really?

Comment (maclover7) [inline spec/paperclip/validators/attachment_size_validator_spec.rb]: I'm :-1: on adding a comma.

File: spec/paperclip/validators/attachment_size_validator_spec.rb [hunk 2]
```diff
@@ -169,8 +163,10 @@ def self.should_not_allow_attachment_file_size(size, options = {})
...
```
Comment (maclover7) [inline spec/paperclip/validators/attachment_size_validator_spec.rb]: I'm :-1: on adding a comma.

**Comment by maclover7** [comment]
Would you be willing to squash your 2 commits down to 1? Otherwise looks ready to merge to me.

**Comment by lucascaton** [comment]
@maclover7 thanks, done!

**Comment by tute** [comment]
Thank you! Merged into master your commit. :)

**Pull request closed without merging**